lightningd/plugin.c

```diff
@@ -215,28 +217,44 @@ static void destroy_plugin(struct plugin *p)
 	}
 }
 
+static u32 file_checksum(const char* path)
+{
+	char *content = grab_file(tmpctx, path);
+	if (content == NULL) return 0;
+	return crc32c(0, content, tal_count(content));
+}
```

Comment:
Well, tal_count(NULL) is 0 so you could skip the second line, but I don't think it matters.

Reply:
Noted. Won't change though, so we don't need to iterate another round of CI and ACKs. Okay?

Reply:
Also It is less clear what crc32c exactly does when called with `len := 0` instead of explicitly `return 0`